openra/tests/routes/test_route_maps_post_comment.py

```diff
@@ -13,7 +13,7 @@ class TestRouteMapsPostComment(TestRouteBase):
     _route = '/maps/{}/post-comment'
 
     def test_route_can_be_accessed_by_any_user(self):
-        user = self.create_old_user();
+        user = self.create_old_user()
```

Comment:
Can these to fixes be squashed back into the first comment?

Reply:
Sure, I've squashed them all up now